ykrt/src/compile/jitc_yk/codegen/x86_64/mod.rs

```diff
@@ -735,6 +765,8 @@ impl<'a> X64CodeGen<'a> {
 
         match (from_size, to_size) {
             (1, 8) => dynasm!(self.asm; movsx Rq(WR0.code()), Rb(WR0.code())),
+            (1, 4) => dynasm!(self.asm; movsx Rd(WR0.code()), Rb(WR0.code())),
```

Comment:
We will have to do something with this match at some point, but not now.

Reply:
I'm not bothered. There aren't that many combinations.